spec/forms/state_file/nj_retirement_warning_form_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe StateFile::NjRetirementWarningForm do
+  let(:intake) {
+    create :state_file_nj_intake,
+    eligibility_retirement_warning_continue: "unfilled"
+  }
+
+  describe "validations" do
+    let(:invalid_params) do
+      { eligibility_retirement_warning_continue: nil }
+    end
+
+    it "requires radio answer" do
+      form = described_class.new(intake, invalid_params)
+      form.valid?
+
+      expect(form.errors[:eligibility_retirement_warning_continue]).to include "Can't be blank."
+    end
+  end
+
+  describe "#save" do
+    context "when all tax household members had health insurance" do
```

Comment:
[pebble] this and the below contexts are for a different form. maybe `when taxpayer chooses to continue` and `when taxpayer chooses not to continue`?